packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testCheckIfClassExists.st

```diff
@@ -0,0 +1,12 @@
+as yet unclassified
+testCheckIfClassExists
+
+	| tree tallyMock compiledMethod |
+	
+	compiledMethod := (BlockClosure >> #ensure:).
+	
+	tree := IPTTreeModelMock new.
+	tallyMock := MessageTallyMock new.
+	tallyMock method: compiledMethod.
+	
+	self assert: true equals: (tree checkIfClassExists: tallyMock).
```

Comment:
You don't need `assert: equals:`, just `assert:` is sufficient:
```smalltalk
self assert: (tree checkIfClassExists: tallyMock)
```
Also you can get rid of the dot `.` in line 12.